webui/react/src/pages/ExperimentDetails/ExperimentVisualization/ExperimentVisualizationFilters.tsx

```diff
@@ -0,0 +1,160 @@
+import { Select } from 'antd';
```

Comment:
A nice to have would be to debounce on changes, eg. when you select/deselect N HP quickly, it will trigger 3 rerenders of charts which is pretty heavy.